Tests/OktaOAuth2Tests/TokenExchangeFlowTests.swift

```diff
@@ -122,6 +131,15 @@ final class TokenExchangeFlowTests: XCTestCase {
         }
 
         XCTAssertFalse(flow.isAuthenticating)
+        
+        XCTAssertEqual(urlSession.requests.count, 3)
+        
+        let request = try XCTUnwrap(urlSession.requests.first(where: { request in
+            request.url?.lastPathComponent == "token"
+        }))
+
+        XCTAssertEqual(request.url?.absoluteString, "https://example.okta.com/oauth2/v1/token")
+        XCTAssertEqual(request.bodyString, "actor_token=secret&actor_token_type=urn:x-oath:params:oauth:token-type:device-secret&audience=api:%2F%2Fdefault&client_id=clientId&grant_type=urn:ietf:params:oauth:grant-type:token-exchange&scope=profile+openid+device_sso&subject_token=id_token&subject_token_type=urn:ietf:params:oauth:token-type:id_token")
     }
     
 #if swift(>=5.5.1) && !os(Linux)
```

Comment:
do we run tests on linux sometimes?

Reply:
We used to have tests running on Linux, but Github Actions' runner images had a problem with that, and I haven't had the impetus to check on that again.  Linux support is currently considered experimental.

Thanks for the reminder though, if I get a few spare cycles, I'll try to test this out.